pkg/api/pullrequestpayloadqualification/v1/types.go

```diff
@@ -101,6 +101,8 @@ type ReleaseJobSpec struct {
 	CIOperatorConfig CIOperatorMetadata `json:"ciOperatorConfig"`
 	// Test is the name of the test in the ci-operator configuration
 	Test string `json:"test"`
+	// AggregatedCount is a number that specifies how many instances of the job will run in parallel
```

Comment:
Let's clarify the corner cases of 0 and 1. It seems that `0` means that the job is not run as aggregated, and `1` would mean an aggregated job with a single execution? If so let's put it into the comment.

Reply:
updated.